coralproject/experiment-playground
# Style toggles with MDL

## What does this PR do?

Converts toggles to MDL and clings with new UX
## How do I test this PR?
- Load playground
- Toggles should be a list of collapsed MDL cards
- Clicking toggles should animate them open
